src/zenml/new/pipelines/run_utils.py

```diff
@@ -241,4 +242,11 @@ def _validate_new_version_requests(
                 "that `Model` requesting new version is configured only in one "
                 "place of the pipeline."
             )
-        data.model._validate_config_in_runtime()
+        is_cloud_model = (
```

Comment:
I know, that it is always True or False, but just for consistency, if this will get changed sometime later,  I'd use `is_cloud_model &= (`

Reply:
thanks, I agree. will push the change